lesson6-questions.txt

```diff
@@ -3,36 +3,44 @@ Questions for Lesson 6
 1. When you enable CRUD operations on a table in the database, you need to add four things
 to your Rails application.  When you generate a scaffold, all four are created.What are they?
 
-
+Models, Views, Contollers, and Routes 
 
 
 2. When you specify a route in config/routes.rb, you need to include certain information.  What
 must be included? In addition, you may specify an additional parameter to create a variable
 that describes the route.  How do you do that? What variable is created?
 
-
+A route includes a verb (GET, POST, PATCH, PUT, DELETE), an URI pattern (path of the URL), and a column for the controller action, and a prefix column 
```

Comment:
> In addition, you may specify an additional parameter to create a variable
that describes the route.  How do you do that? What variable is created?

That is an alias, like `get 'users/:id', to: 'users#show', as: user`